src/building/how-to-build-and-run.md

```diff
@@ -54,6 +54,12 @@ clean` will not cause a rebuild of LLVM.
 
 ## Building the Compiler
 
+Building compiler is unfortunately not quite as simple as building other
+rust projects using `cargo build`. Instead, we use `x.py` script for building
+the compiler. The main reason behind it is the bootstrap stages where you use
+the output of one cargo invocation as the input compiler for another. Which
+is not possible with `cargo` itself.
```

Comment:
We already talk about x.py above at the very top. I think it's ok to just delete this paragraph altogether.